manifests/config.pp

```diff
@@ -74,13 +26,15 @@
   }
 
   if $::osfamily == 'RedHat' {
-    file { '/etc/oddjobd.conf.d/oddjobd-mkhomedir.conf':
-      ensure  => 'file',
-      owner   => 'root',
-      group   => 'root',
-      mode    => '0644',
-      content => template('winbind/oddjobd-mkhomedir.conf.erb'),
-      notify  => Service[['oddjobd', 'winbind',]],
+    if ($winbind::manage_oddjob_service == true) {
```

Comment:
Ok, that was a misunderstanding on my part then about that file. So should an else block be added with the same file resource except it contains only a notify for the winbind service instead of both oddjobd AND winbind?

FYI this problem originated because if `manage_oddjob_service` is false, then test fails because the `oddjobd` service is not contained in the catalog:

https://github.com/genebean/genebean-winbind/blob/3e6a975759c75cf58264a120b7a63e9fa705b597/manifests/service.pp#L15